packages/eslint-plugin-result/src/configs/recommended.ts

```diff
@@ -0,0 +1,5 @@
+export const recommendedConfig = {
+	rules: {
+		'@sapphire/result/no-discard-result': 'error'
+	}
```

Comment:
```suggestion
    plugins: ['@sapphire/eslint-plugin-result'],
	rules: {
		'@sapphire/result/no-discard-result': 'error'
	}
```

The point really of a recommended config is to include the plugin itself, many other plugins do it like this too. Eslint automatically merges the plugins array.